.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 services:
 - rabbitmq
 node_js:
-- '0.10'
+- '4.2'
```

Comment:
This breaks autodeploy due to NPM version/keyformat change, so you will need to update that as well.